sdks/java/core/src/main/java/org/apache/beam/sdk/Pipeline.java

```diff
@@ -30,6 +31,8 @@
 import javax.annotation.Nonnull;
 import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.coders.CoderRegistry;
+import org.apache.beam.sdk.errorhandling.ErrorHandler;
```

Comment:
would it be reasonable to put errorhandling under org.apache.beam.sdk.transforms? Since this is a PTransform feature. Some other features (splittabledofn, resourcehints) are also in this package.

Reply:
I agree, it makes sense to put there